hypervisor/vm.go

```diff
@@ -322,6 +322,7 @@ func (vm *Vm) StopPod(mypod *PodStatus, stopVm string) *types.VmResponse {
 	}
 
 	mypod.Status = types.S_POD_FAILED
+	mypod.FinishedAt = time.Now().Format("2006-01-02T15:04:05Z")
```

Comment:
Hmm, 

Anyway, you are correct

https://github.com/hyperhq/hyperd/blob/master/daemon/pod.go#L1007

Here should be a bug --

if the vm exited, no matter it is terminated by user or the process finished itself, it should call `PodStatus.SetPodContainerStatus()` instead of `SetContainerStatus`